services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -870,6 +465,69 @@ impl NetCdfCfDataProvider {
             band_offset: dataset_id.entity as usize * dimensions.time,
         }))
     }
+
+    fn meta_data_from_overviews(
+        overview_path: &Path,
+        dataset_id: &NetCdfCf4DDatasetId,
+    ) -> Option<GdalMetadataNetCdfCf> {
+        let loading_info_path = overview_path
+            .join(&dataset_id.file_name)
+            .join(&dataset_id.group_names.join("/"))
+            .join("ebv_cube.json");
+
+        let loading_info_file = std::fs::File::open(&loading_info_path).ok()?;
+        let mut loading_info: GdalMetadataNetCdfCf =
+            serde_json::from_reader(BufReader::new(loading_info_file)).ok()?;
+
+        // it is 1 + … because we have one step when start == end
+        let time_steps_per_entity = 1 + loading_info
+            .step
+            .num_steps_in_interval(TimeInterval::new(loading_info.start, loading_info.end).ok()?)
+            .ok()?;
+
+        // change start band wrt. entity
+        loading_info.band_offset = dataset_id.entity * time_steps_per_entity as usize;
```

Comment:
does this still have to be manipulated if it is loaded from an overview?

Reply:
Sure, for GDAL it is the same, having the Tiff with lots of bands than having the NetCDF cube.